rhcos-bfb.Containerfile

```diff
@@ -163,8 +158,8 @@ RUN dnf -y install --setopt=install_weak_deps=False \
   vim-common \
   dhcp-client \
   && dnf clean all && \
-  rpm -e --nodeps libnl3-devel kernel-headers libzstd-devel ncurses-devel libpcap-devel \
-  elfutils-libelf-devel meson libyaml-devel ninja-build epel-release
+  rpm -e --nodeps ngauge spdk collectx-clxapi doca-dms libnl3-devel kernel-headers libzstd-devel ncurses-devel libpcap-devel \
```

Comment:
For some reason something else seemed to drag spdk into my build - but we don't need it.